kitsu/background.js

```diff
@@ -0,0 +1,9 @@
+chrome.tabs.onUpdated.addListener(function (tabId, changeInfo, tab) {
+    if (changeInfo && changeInfo.status == 'complete' && tab.url.match("/kitsu.io/anime/")) {
+        inject(tabId);
+    }
+});
+
+function inject(tabId) {
+    chrome.tabs.executeScript(tabId, { file: "kitsu/script.js" })
+}
```

Comment:
unnecessary file

Reply:
This is not unnecessary. Without this, if you go on the Kitsu homepage and click on any anime, it doesn't work, leaving the only way it works being you having to type the anime's URL in the browser yourself or having to reload the page again after going there from the homepage.

Reply:
oh okay didn't notice it has dynamic loading. okay, let this stay.